storage.go

```diff
@@ -132,12 +132,10 @@ func (s *Storage) createDir(ctx context.Context, path string, opt pairStorageCre
 	return o, nil
 }
 
-func (s *Storage) createLink(ctx context.Context, path string, target string, opt pairStorageCreateLink) (o *Object, err error) {
-	if !s.features.VirtualLink {
-		err = NewOperationNotImplementedError("virtual_link")
-		return nil, err
-	}
+// LinkMeta is the name of the user-defined metadata name used to store the target.
+const LinkMeta = "x-amz-meta-bs-link-target"
```

Comment:
How about keeping this header key private?

How about names like `metadataLinkTargetHeader`?

Reply:
Got it!